internal/errgroup/group.go

```diff
@@ -38,6 +39,9 @@ type group struct {
 
 	wg sync.WaitGroup
 
+	esctx  context.Context
+	escape func()
```

Comment:
```suggestion
	escape context.CancelFunc
```
also, could you please fix line 38?